content/en/blog/falco-pci-dss/index.md

```diff
@@ -0,0 +1,165 @@
+
+By continuously monitoring containerized environments, Falco can detect security events, which helps engineering teams enforce secure configurations, identify potential vulnerabilities, and provide real-time insights into system and application behavior. Its ability to generate real-time alerts, trigger automated responses, and integrate with other security tools empowers organizations to develop and maintain secure systems and applications that align with PCI requirements.
+
+### PCI Sub-Control #6.4.2
+
+**Separation of duties between development/test environments**
+
+The Falco rule "[Network Connection Outside Local Subnet](https://github.com/falcosecurity/rules/blob/c558fc7d2d02cc2c2edc968fe5770d544f1a9d55/rules/falco_rules.yaml#L2962)," is an excellent example of how Falco can be utilized to detect the separation of duties between development/test environments for containerized workloads, specifically in the context of PCI control 6.4.2.
+
+The [PCI control 6.4.2](https://security.stackexchange.com/questions/12962/pci-dss-requirement-6-4-2-separation-of-duties-between-development-test-environm) focuses on ensuring that development/test environments are segregated from production environments in order to minimize the risk of unauthorized access or data leakage. By monitoring network connections within containerized environments, Falco can play a crucial role in enforcing this control.
+
+The local subnet network connection rule is designed to alert whenever a container attempts to establish unwarranted network connections. This indicates that a container in the development/test environment is trying to communicate with resources outside its designated network boundaries. Such communication might be a sign of misconfiguration, unauthorized access attempts, or a potential data exfiltration attempt.
+
+When Falco detects a network connection outside the local subnet, it generates an alert or takes a defined action, such as logging the event, sending notifications to security personnel, or triggering automated responses. This allows security teams to identify and investigate any suspicious activities that violate the separation of duties between development/test and production environments.
+
+```
+- rule: Launch Privileged Container
+  desc: Detect the initial process started in a privileged container.
+        Exceptions are made for known trusted images.
+  condition: >
+    container_started and container
+    and container.privileged=true
+    and not falco_privileged_containers
+    and not user_privileged_containers
+    and not redhat_image
+  output: Privileged container started (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline)
+  priority: INFO
+  tags: [container, privilege_escalation, lateral_movement, T1610, PCI_DSS_10.2.5]
```

Comment:
@nigeldouglas-itcarlow oh I love this convention about the PCI/DSS reference, can we learn more it, what standards did you base it on (I am more of a threat detection and less compliance person). We are discussing a new rules maturity and adoption framework here https://github.com/falcosecurity/rules/pull/76 and we wanted to add at least 3 "compliance" rules by Falco 0.36 and/or tag existing ones in this regard? Could you help us? Any feedback welcome re the rules maturity framework proposal!

Reply:
@incertum I am happy to help with this. I only tagged the existing rules with the relevant PCI DSS tags based on the standards/controls outlined in the PCI compliance framework - https://www.pcidssguide.com/pci-dss-requirement-10/

I am also working on aligning some existing Falco rules with NIST 800-171 Controls. If this helps?
Either way, I'd be happy to discuss this offline. Is there a meeting set up for the maturity framework proposal?